### ISSUE TITLE
Recovery of deleted files in/w dupeguru

### ISSUE THREAD

**Issue by HelpPlease9** [arsenetar/dupeguru#987]
I know nothing about coding. I have a Win 10 PC. I used dupeguru to scan my whole HD. I created a folder on my EX HD to save the files. Now, once the scan is done, I cannot open File Explorer. That being the case, I am looking for a way via dupeguru to restore all of my files. How can I do this? Thanks. PS: I thought of going to my 'restore point' with the Win 10 OS - (I created a restore point just before I did the scan) but I do not know if that would fix the problem or make it worse.

**Comment by arsenetar** [comment]
Closing as this is a duplicate of a discussion thread.
